src/appleseed.shaders/CMakeLists.txt

```diff
@@ -403,6 +403,7 @@ foreach (osl_source ${osl_sources})
         OUTPUT ${oso_dir}/${oso_filename}
         COMMAND ${CMAKE_COMMAND} -E make_directory ${oso_dir}
         COMMAND ${PROJECT_SOURCE_DIR}/sandbox/bin/oslc ${OSLC_OPTIONS}
+            -I"${PROJECT_SOURCE_DIR}/sandbox/shaders/"
```

Comment:
OSL 1.11.7 and 1.11.8 changed where `stdosl.h` is stored (PREFIX/share/shaders instead of ../shaders), and its location is stored as a compile time constant as an absolute path (not relative).
However, if `stdosl.h` found in the includes it takes precedence.
These changes were introduced in https://github.com/imageworks/OpenShadingLanguage/pull/956 and the relevant change can be seen here:
https://github.com/imageworks/OpenShadingLanguage/blob/8ff28a0e062c714b74244cc89fd55461421c5a97/src/liboslcomp/oslcomp.cpp#L334-L341